services/currentPantry.js

```diff
@@ -44,10 +44,13 @@ const getPantryItemByName = name => getDbConn(dbAddr).any(
 const getPantryItemsOfUser = id => getDbConn(dbAddr).any(
```

Comment:
rename this `id` to `userId` or `ownerId` so you know that its not the primary key of the `currentPantry` table.

Reply:
Perfect, will change it!